src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,90 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block a role from the command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: '876543211234567890',
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
+  async addPermissions(command, permissions) {
+    const id = this.resolveID(command);
+    if (!id) throw new TypeError('INVALID_TYPE', 'command', 'ApplicationCommandResolvable');
+
+    const perms = await this.fetchPermissions(id).catch(err => {
+      if (err.httpStatus !== 404) throw err;
+      return [];
+    });
+
+    const newPermissions = [...permissions];
```

Comment:
```suggestion
    const newPermissions = permissions.slice();
```
probably not much of a difference but i remember this being used somewhere